tools/pipelines/templates/build-npm-package.yml

```diff
@@ -70,7 +70,7 @@ parameters:
 
 - name: nonScopedPackages
   type: object
-  default: []
+  default:
```

Comment:
nit: did you mean to leave this empty?

Reply:
Yes, but I should just revert this part of the change, because it's unnecessary.